docs/FallbackPackageCache.md

```diff
@@ -0,0 +1,48 @@
+Fallback Package Cache (LZMA)
+=============================
+
+The fallback package cache (commonly called the LZMA) is a set of NuGet packages that are bundled in the .NET Core SDK installers.
+The LZMA is a compressed file format, similar to a .zip.
+On first use, the .NET Core CLI will expand this LZMA file, extracting the packages inside to %DOTNET_ROOT%/sdk/NuGetFallbackFolder,
+or %USERPROFILE%/.dotnet/sdk/NuGetFallbcakFolder, if %DOTNET_ROOT% is not writable (i.e. C:/Program Files/dotnet).
```

Comment:
We don't do this. We always extract it next to the SDK itself.

Reply:
Oh, what happens if C:/Program Files/dotnet isn't writable?

Reply:
You get an error saying so with a couple of alternatives on what you can do. However, usually, when you install to C:\Program Files\dotnet, that is through the MSI and the MSI expands the cache as part of the install process.

Reply:
Cool. TIL how that works. Updated